### DISCUSSION TITLE
FluxStore

### DISCUSSION THREAD

**Discussion by Moeed366** [inspireui/support#5786] [New Feature/Ideas Requests] [answered]
Hi. I have purchased Fluxstore Shopify. I have provided the purchased key to the code but got an error on the app loading it crashed and gave an error Please provide your purchase code in configs/env.properties.

**Comment by chungxon** [comment, marked as answer]
Hi mate @Moeed366,
Please help submit a ticket to our support center https://support.inspireui.com/help-center
Thank you!
